antora.yml

```diff
@@ -13,4 +13,97 @@ nav:
+    identity-provider-id: keycloak
+    identity-provider-image-url: quay.io/eclipse/che-keycloak
+    identity-provider-url: https://www.keycloak.org/
+    identity-provider-version: "6.0"
+    identity-provider: Keycloak
+    image-puller-cr-name: KubernetesImagePuller
+    image-puller-deployment-id: kubernetes-image-puller-operator
+    image-puller-deployment-name: kubernetes-image-puller
+    image-puller-image-name: quay.io/eclipse/kubernetes-image-puller
+    image-puller-image-tag: latest
+    image-puller-images: "java11-maven=quay.io/eclipse/che-java11-maven:nightly;che-theia=quay.io/eclipse/che-theia:next;java-plugin-runner=eclipse/che-remote-plugin-runner-java8:latest;"
+    image-puller-name-short: Image Puller
+    image-puller-name: Kubernetes Image Puller
+    image-puller-namespace: k8s-image-puller
+    image-puller-operator-id: kubernetes-image-puller-operator
+    image-puller-operator-name: Kubernetes Image Puller Operator
+    image-puller-operator-repository-name: kubernetes-image-puller-operator
+    image-puller-repository-name: kubernetes-image-puller
+    kube-ver-min: "1.19"
+    kubernetes: Kubernetes
+    link-accessing-a-git-repository-via-https: xref:end-user-guide:version-control.adoc#accessing-a-git-repository-via-https_che[Accessing a Git repository using HTTPS]
+    link-advanced-configuration-options-for-the-che-server: xref:installation-guide:advanced-configuration-options-for-the-che-server-component.adoc[]
+    link-advanced-configuration-options: xref:installation-guide:advanced-configuration-options-for-the-che-server-component.adoc[]
+    link-building-a-custom-plug-in-registry: xref:administration-guide:building-and-running-a-custom-registry-image.adoc#building-a-custom-devfile-registry_che[Building a custom devfile registry]
+    link-cli-github: https://github.com/che-incubator/chectl
+    link-identity-provider-documentation-openshift-3: https://www.keycloak.org/docs/latest/server_admin/#openshift-3
+    link-identity-provider-documentation-openshift-4: https://www.keycloak.org/docs/latest/server_admin/#openshift-4
+    link-identity-provider-github: "https://www.keycloak.org/docs/6.0//server_admin/#github"
+    link-identity-provider-oidc: "https://www.keycloak.org/docs/6.0/server_admin/#_oidc"
+    link-identity-provider-saml: "https://www.keycloak.org/docs/6.0/server_admin/#saml-v2-0-identity-providers"
+    link-installing-an-instance: xref:installation-guide:installing-che.adoc[]
+    link-postgres-dockerfile-location: https://github.com/eclipse-che/che-server/tree/main/dockerfiles/postgres
+    link-server-identity-provider-dockerfile-location: https://github.com/eclipse-che/che-server/tree/main/dockerfiles/keycloak
+    link-viewing-the-state-of-the-cluster-deployment-using-openshift-4-cli-tools: xref:overview:installing-che-on-openshift-4-using-operatorhub.adoc[]
+    namespace: namespace # In context: API namespace
+    nodejs-stack: nodejs
+    oc4-ver: "4.6"
```

Comment:
this is minimum ocp version, I assume? we also support 4.8 (and 2.12 will likely support 4.9)

Reply:
This attribute is never used. Removing it.
```suggestion
```